src/components/Page/components/Header/components/Title/Title.tsx

```diff
@@ -56,7 +56,7 @@ export function Title({title, subtitle, titleMetadata, thumbnail}: TitleProps) {
 
   const subtitleMarkup = subtitle ? (
     <div className={styles.SubTitle}>
-      <p>{subtitle}</p>
+      {typeof subtitle === 'string' ? <p>{subtitle}</p> : <div>{subtitle}</div>}
```

Comment:
To maintain the existing behavior I went with this approach but I'm more than happy to switch to using `div` for all cases.